code/math/vecmat.h

```diff
@@ -22,9 +22,15 @@
 //Macros/functions to fill in fields of structures
 
 //macro to check if vector is zero
```

Comment:
Can you change this comment to match what `IS_VEC_NULL_SQ_SAFE` is specifically for, and then probably move this comment to be above `IS_VEC_NULL`?

Reply:
Sure, the original code had it this way but happy to take the opportunity to update comments.

Reply:
The original macro split happened back in 2009: 75a514b5f9

I would be curious if anyone has a good idea how these thresholds should be determined.  The well-known `epsilon` is for addition and subtraction; I would assume there is some other standard for multiplication and squaring.

Reply:
My theory is that the `SQ_SAFE` version ensures that squaring a float is safe. But I don't have evidence for that beyond common sense.

Reply:
I think the existing epsilon is fine. This issue is MOI-specific, imo. Due to the way they are calculated and used, they are unfortunately extremely small numbers a lot of the time.

Reply:
Makes sense. Are folks good with the current version then?

Reply:
Yeah, go ahead